### DISCUSSION TITLE
Does map_local prevent the request from being made to the target host?

### DISCUSSION THREAD

**Discussion by kellynichols** [mitmproxy/mitmproxy#5114] [Q&A] [answered]
I'm looking at using mitmproxy for mocking requests during local development workflows. Does map_local actually prevent filtered requests from hitting the target hosts? Or are requests made as usual and the response body is simply read from local files? Thanks!

**Comment by mhils** [comment, marked as answer]
Filtered requests will not hit the target host. :)

**Comment by kellynichols** [comment]
@mhils I figure this was the case but just wanted to confirm. Many thanks!
